pkg/util/spanlogger/spanlogger.go

```diff
@@ -34,6 +39,9 @@ func New(ctx context.Context, method string, kvps ...interface{}) (*SpanLogger,
 // retrieved with FromContext or FromContextWithFallback.
 func NewWithLogger(ctx context.Context, l log.Logger, method string, kvps ...interface{}) (*SpanLogger, context.Context) {
 	span, ctx := opentracing.StartSpanFromContext(ctx, method)
+	if ids, _ := tenant.TenantIDs(ctx); ids != nil {
```

Comment:
instead of `ids != nil` i'd check `len(ids) > 0`, because if `ids == nil` then `len(ids) == 0` but `ids` could also be `[]string{}` which is `!= nil`.
```suggestion
	if ids, _ := tenant.TenantIDs(ctx); len(ids) > 0 {
```

https://play.golang.org/p/zO8541dX4xN

Reply:
done